### DISCUSSION TITLE
Is it posible to find station reports via API endpoint?

### DISCUSSION THREAD

**Discussion by jonasjersborg** [AzuraCast/AzuraCast#7151] [Q&A] [answered]
When i use the admin i can use a report for station statistics, but i dont see it in the API?
https://{url}/api/station/2/reports/overview/best-and-worst?start=2024-05-14T23:07:31.868%2B02:00&end=2024-05-27T23:07:31.868%2B02:00
If i try to use the url in Postman, i get a response that im not logged in?

**Comment by Muta-Jonathan** [comment]
@jonasjersborg try to add an API KEY in your azuracast 
https://www.azuracast.com/docs/developers/apis/#api-authentication
After in postman add **`Authorization: Bearer {your_api_key_here}`** as an header

**Reply by jonasjersborg** [reply]
Thanks :) But i already use the other endpoints. My question is why endpoint /reports/ is not a part of the API set

**Comment by BusterNeece** [comment, marked as answer]
The report API endpoints are considered "internal" (thus we don't necessarily try to keep them the same from version to version the way we do with our documented endpoints), but you should certainly be able to call them the same way you'd call any of the other API endpoints as long as you have authorization with a key with the appropriate permissions.

**Reply by jonasjersborg** [reply]
Thanks Buster! 
It would be great if the API documentation showed the internal endpoints or that you made them external.
Azuracast keeps getting better and better, so I hope to integrate more into our backend.
Great work 💯
